.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
```

Comment:
What is husky?

Reply:
Looks like that script is not in the repo?

Reply:
husky is a tool to set up git hooks with prettier. So if you try to commit a file with unformatted code, it will correct the formatting first.
The script is generated after install. It is gitignored.